### DISCUSSION TITLE
When will WebView2 be supported on Linux?

### DISCUSSION THREAD

**Discussion by sam-wheat** [unoplatform/uno#12041] [General]
http://unoplatformdocstaging.z13.web.core.windows.net/articles/controls/WebView.html

> WebView2 is currently supported on Windows, Android, iOS and macOS.

**Comment by jeromelaban** [comment]
We're currently tracking the work for this in https://github.com/unoplatform/uno/issues/4233. We do not have an ETA for this feature.
